redis.conf

```diff
@@ -1702,5 +1702,20 @@ rdb-save-incremental-fsync yes
 # active-defrag-max-scan-fields 1000
 
 ############################### PMEM CONFIG ###############################
+
+# Redis supports four different Persistent Memory policies:
```

Comment:
I suggest calling it more general: Memory Allocation Policy instead of Persistent Memory Policy.

Reply:
Done. Also, I extend the description in redis.conf